lucille-plugins/lucille-api/src/test/java/APIIntegrationTest.java

```diff
@@ -0,0 +1,47 @@
+import static org.junit.Assert.assertEquals;
+
+import com.kmwllc.lucille.APIApplication;
+import com.kmwllc.lucille.LucilleAPIConfiguration;
+import io.dropwizard.testing.ResourceHelpers;
+import io.dropwizard.testing.junit.DropwizardAppRule;
+import jakarta.ws.rs.client.Client;
+import jakarta.ws.rs.core.Response;
+import java.io.ByteArrayInputStream;
+import java.nio.charset.StandardCharsets;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+public class APIIntegrationTest {
+  // TODO : Improve Integration testing with better mocking
```

Comment:
is there something specific we want to do here re: improvement? if so, let's create a ticket. 
suggest removing the TODO comment

Reply:
I don't think there are any further improvements required anymore since the tests were switched to use a test config. The TODO has been removed.